app/views/includes/global_error.scala.html

```diff
@@ -16,40 +16,38 @@
 
 @(globalErrorCode: Integer)(implicit messages: Messages, application:config.ApplicationConfig)
 
-
-
 @queryHandlingHeader = {
     @if(application.globalErrors.hasPath(s"$globalErrorCode.solution")){
-        <h2 class="error-h2" id="global-query-handling-header">@{ Messages("gmp.errors.query_handling.header") }</h2>
+        <h2 class="govuk-heading-l" id="global-query-handling-header">@{ Messages("gmp.errors.query_handling.header") }</h2>
     }
 }
 
 <div id="global-results-grid">
 
         <div class="global-error">
             @if(application.globalErrors.hasPath(s"$globalErrorCode.cannot_calculate")){
-                <h1 class="error-h1" id="cannot-calculate-header">@{ Messages(application.globalErrors.getString(s"$globalErrorCode.cannot_calculate")) }</h1>
+                <h1 class="govuk-heading-l" id="cannot-calculate-header">@{ Messages(application.globalErrors.getString(s"$globalErrorCode.cannot_calculate")) }</h1>
             }
 
             @if(application.globalErrors.hasPath(s"$globalErrorCode.reason")){
-                <div class="error-list" id="global-error-reason">@Messages(application.globalErrors.getString(s"$globalErrorCode.reason"))</div>
+                <div class="error-list" id="global-error-reason"><p class="govuk-error-message">@Messages(application.globalErrors.getString(s"$globalErrorCode.reason"))</p></div>
```

Comment:
Should this be "govuk-error-summary__list" or "govuk-list" instead of the old error-list

Reply:
govuk-error-summary__list expects a href as a parameter, and then converts it into red colour. but in this scenario, it was just a message with red color and no hyperlink  , so kept it as it is. If the design needs to be changed for this we need confirmation from Daren or jenny